md/pages.md

```diff
@@ -95,13 +95,21 @@ out.write("""<iframe src=../portal.js/#/admin/monitoring/${idProcess}-${idCase}?
 
 While you are developing a custom page, you can enable custom page debug mode for your portal. In debug mode, you can see changes to your custom page without importing a new zip archive.
 
+::: info
+Before use setup tool for the studio context, you need to edit the file : `database.properties`, and re-define `h2.database.dir=../../<yourRepository>/h2_database`  
```

Comment:
1- This is still weird to talk about setup tool here. You must at least explain why this file change is needed.
And link to the setup tool doc page to make it clearer.

2- you say `Before use setup tool...` but then you don't talk about setup tool. Something is missing...